Components/WideImage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
```

Comment:
Prettier will do this -- Make sure you either have all of your imports on one line or one line per import from any given module.